maintainers/maintainer-list.nix

```diff
@@ -6512,6 +6512,11 @@
     githubId = 1817528;
     name = "Igor Polyakov";
   };
+  iquerejeta = {
+    name = "Inigo Querejeta-Azurmendi";
```

Comment:
We also try to have attributes sorted alphabetically, but it's just a detail. If you modify this once again, please sort them.